src/utils/serialize-query-params.js

```diff
@@ -1,17 +1,17 @@
-import { isPlainObject } from 'ember-fetch/types';
-
 const RBRACKET = /\[\]$/;
 
+function isPlainObject(obj) {
+  return Object.prototype.toString.call(obj) === '[object Object]';
+}
+
 /**
  * Helper function that turns the data/body of a request into a query param string.
  * This is directly copied from jQuery.param.
```

Comment:
There's also https://github.com/knowledgecode/jquery-param, so wondering if we should still include this in ember-fetch?